packages/deploy-tooling/src/base/deploy.ts

```diff
@@ -310,7 +315,7 @@ async function tryDeploy(
                 'Deployment in TestMode completed. A successful TestMode execution does not necessarily mean that your upload will be successful.'
             );
         } else {
-            logger.info('Successfully deployed.');
+            logger.info('Deployment Successful.');
```

Comment:
There was also an ask to revert to this statement, I can move to a separate PR if required but as it is so small ...